docs/js-core-1/week-2/lesson.md

```diff
@@ -429,6 +430,64 @@ So to sum up, `for of` loops are pretty similar to `for` loops, but:
 
 Next week, we will think more about the most common iterable object in JavaScript: [arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).
 
+## PD (Dealing with conflict in teams)
+
+Adapted from [Tectonica Interpersonal Conflict Resolution](https://github.com/Techtonica/curriculum/blob/main/career/conflict-resolution.md).
+
+**Session objective:** This session explores what conflict is and the common ways we address it. We will explore conflict resolution toolkits and how we can become better at managing conflict effectively. We will talk about what effective body language is; what effective choice of words is; and how to build trust among teams.
+
+[Conflict resolution slides](https://docs.google.com/presentation/d/1hTBEMDXrYBC2QTXWaRJkWI378DTVF-s8t_Uxi8mT7bQ/edit#slide=id.g1b73cb08c45_0_0)
+
+
+:::note Exercise 1 (15 minutes)
+
+**Exercise objective:** To role-play a workplace conflict scenario.
+
+We are going to read through the scenario and practice with some role play.
+
+In groups of 3 – one person will play the role of Missy, one person will play her manager, and the third person will be the observer.
+
+Missy’s:  It’s after the meeting you are going to talk with your manager about what happened.  You want to practice doing this the right way – click back to previous slide for advice.  
```

Comment:
I think we need an AND or a comma.

It’s after the meeting **and** you are going to talk with your manager about what happened.